Adding filter to ignore all files that starts with a dot

DerekFangming opened the pull request:
This PR is for the problem discussed with @drallgood at #100 

[Here's the problematic pass file](https://drive.google.com/file/d/1OR_wbf7UT_hcO643yrc61G8ROkyV4Wxo/view?usp=sharing) if you still need to see it.

### **Why the pass failed validation and cannot open:**
After clearly reviewing apple's signpass project, which I believe is also the validation code before a pass is actual opened, I found this piece of code within PassSigner.m
```
NSArray *argsArray = [NSArray arrayWithObjects:@"-q", @"-o", [passURL path], @"-d", [tempURL path], nil];
PSPrintLine(@"\n*** %@", [passURL path]);
PSPrintLine(@"\n*** %@", [tempURL path]);
[unzipTask setArguments:argsArray];
[unzipTask launch];
[unzipTask waitUntilExit];
```
So basically, they are unzipping the pkpass file with command:
```

unzip -q -o <PathToPkpassFile> -d <PathToDestinationFolder>
```
When I run this command on the attached pkpass file, all files starting with the dot are gone. Then later in the validation process, those files appear to be missing because manifest file has them.
Try to unzip the file yourself and you can see that these file starting with dot are actually inside.

### **Where does these files come from:**
My server is running OSX. Files starting with ._ are files that the OS uses to keep some meta data, etc. These files do no exist in my source folder. I believe that they are created when jpasskit copies the template folder using FileUtils.

### **The fix:**
Ignore all files starting with dot. I have reviewed pkpass documents and none of the pass template has file starting with dot. Also, we don't want to put stuffs like .DS_Store in the pass file anyway.

### **I tried the code and now I can open the ticket file 👍  **

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/drallgood/jpasskit?pullRequest=115) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/drallgood/jpasskit?pullRequest=115) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/drallgood/jpasskit?pullRequest=115) it.</sub>

drallgood commented:
@DerekFangming Thanks for submitting this PR.

Could you please try to add a few tests for this and also update the CHANGELOG file?

Thanks!

DerekFangming commented:
@drallgood Added test cases.
Have to do this in this approach. I added two files starting with dot.

For PKFileBasedSigningUtil, added asserts to make sure that dot file is not included.
For the other two places, you already have asserts to see if file count is 8. And they are not 10 after the two files I added

drallgood commented:
@DerekFangming Awesome. Thank you so much for fixing this!
